doctrinal-works-v3/original/03-chapter-3.adoc

```diff
@@ -540,13 +540,16 @@ and all his holy prophets and apostles, and all people upon the earth,
 for their salvation, and desires their eternal good and happiness,
 and not any man`'s destruction.
 
+[.signed-section-signature]
 G+++.+++ Fox.
 
+[.signed-section-context-close]
 Kingston upon Thames, the 16th of the fifth Month, 1680.
 
 [.old-style]
```

Comment:
I tried to embed this but the title didn't work right. But it seemed to be in its own section so maybe not embedding is okay?

Reply:
i think not embedding it is fine. 👍